regression/ansi-c/static2/main2.c

```diff
@@ -1,4 +1,4 @@
 static int foo(int a)
 {
-  return a+1;
+  return a + 2;
```

Comment:
why?

Reply:
This makes sure the functions are actually different so that the `static` variant doesn't get ignored after the changes in this PR.